src/pages/gamer/[platform]/[username].tsx

```diff
@@ -65,7 +65,6 @@ export default function GamerDetail({gamerData, view}) {
                                     onClick={() => setTabAndFetchData(gamer.username, gamer.platform, "teammates", HOSTNAME, chartState, setChartState)}>Teammates</Button>
                                 <Button
                                     onClick={() => setTabAndFetchData(gamer.username, gamer.platform, "placements", HOSTNAME, chartState, setChartState)}>Placements</Button>
```

Comment:
Quick question here.. Why are we passing in gamer.username, gamer.platform, HOSTNAME, chartState, setChartState each time this is set? Wouldn't these things be available in "this" from the class based component? is this a function of using functions?

and we really ought to just group this into 2 parameters: viewName & config ({username, platform, chartState, setChartState})

Config could be set beforehand up above so we don't have to re-type it